modules/programs/tmux.nix

```diff
@@ -199,10 +453,10 @@ in
 
       shortcut = mkOption {
         default = defaultShortcut;
```

Comment:
Seems to me that this will make the documentation depend on the current configuration. Perhaps use `defaultText` to avoid this?